### ISSUE TITLE
CornerRadius not working in Xamarin.iOS current versions (somewhat workaround inside)

### ISSUE THREAD

**Issue by idchlife** [robinmanuelthiel/flexbutton#91]
**Describe the bug**
I updated Xamarin Forms and CornerRadius no longer works.

**To Reproduce**
*Steps to reproduce the behavior:*
1. Update any project or create new with versions:

Xamarin.iOS - 13.20.1.18
Xamarin.Forms- 4.7.0.1080

2. Run iOS project. See that buttons is not rounded if you specify CornerRadius.

**Expected behavior**
CornerRadius to work. Buttons square as if there was no corner radius.

**Screenshots**

**Please complete the following information:**
- Which version of the FlexButton do you use? - **0.11.0**
- Which version of Xamarin.Forms do you use? 4.7.0.1080
- Which OS are you talking about? iOS

**Additional information if needed:**

I fixed it in my app by providing custom ViewRenderer for iOS with this code:

```cs
[assembly: ExportRenderer(typeof(FlexButton), typeof(FlexButtonFixRenderer))]
namespace ReceiptsApp.iOS.Views.UI {
  public class FlexButtonFixRenderer : ViewRenderer {
    public FlexButtonFixRenderer() {
    }

    protected override void OnElementChanged(ElementChangedEventArgs<View> e) {
      base.OnElementChanged(e);

      Debug.WriteLine("IM HERE");

      if (e.NewElement == null) return;

      var element = e.NewElement;

      if (element == null) return;

      Debug.WriteLine("ELEMENT IS NOT NULL!");

      try {
        if (Element is FlexButton flex) {
          Debug.WriteLine("FIXING FLEX BUTTON BUG WITH NO CORNER RADIUS");
          if (flex.CornerRadius <= 0) return;

          NativeView.Layer.CornerRadius = flex.CornerRadius;
          //NativeView.ClipsToBounds = true;

          Debug.WriteLine("DONE FIXING");
          //control.Layer.Boun
	  
        }
      } catch (Exception exc) {
        Debug.WriteLine("ERROR FIXING FLEX BUTTON ROUND EFFECT");
        Debug.WriteLine(exc);
      }

    }
  }
}
```

BUT it kills border because of ClipsToBounds 😩

**Comment by mike-rowley** [comment]
@idchlife I think this project has been abandoned.  I posted a message over 6 weeks ago asking Robin, the repository owner, about its status and haven't received a response.

I have pulled the code repository and am making necessary changes within my own project code base, unfortunately I don't have the time or requisite control building experience to continue development on this publicly.

I have run into the issue you have described and thought I would share what I have found, it appears to be related to this Xam Forms bug:
https://github.com/xamarin/Xamarin.Forms/issues/2405
